src/Inkluzitron/appsettings.json

```diff
@@ -5,8 +5,10 @@
     "Spongebob": "<a:mocking:837629478792200213>",
     "Hugging": "<:peepohug:753635400488321144>",
     "PepeHands": "<:pepehands:834159416269864990>",
-    "CrackTippingEmote": "<:mcracko:835901351284310026>",
+    "CrackTippingEmote": "<:tipfedora:870263274606854164>",
+    "NewbieInviteMessage": "Jako nováček bohužel nemůžeš sám někoho dalšího pozvat.\nPokud sem chceš někoho pozvat, napiš do <#834127024385425450> a tam se o tom všichni pobavíme <:peepolove:753635400723202065>",
```

Comment:
Přemýšlím, zda je třeba sem uvádět to slovo "bohužel". Vzhledem k tomu, že to bude chtěný stav pro naši "bezpečnost".